### ISSUE TITLE
VGrive permanently deleted half of my Google Drive Directory 

### ISSUE THREAD

**Issue by thankala** [bcedu/VGrive#75]
VGrive actually permanently deleted half of my Google Drive Directory (GSuite for Edu Account). How is this possible?
**Screenshots**

![Screenshot_20200119_192457](https://user-images.githubusercontent.com/43992756/72685322-c67f9200-3af1-11ea-8f98-f8cca5b62366.png)
![Screenshot_20200119_192524](https://user-images.githubusercontent.com/43992756/72685325-ced7cd00-3af1-11ea-9b3a-9b09095bb3ef.png)

**Desktop :**
 - OS: Manjaro KDE
 - VGrive Version v1.4 installed from AUR

**Comment by thankala** [comment]
I did find everything on .trash folder but why cant they be simple move to trash bin in gdrive?

**Comment by tghgg** [comment]
Vgrive deleted some of my important stuff as well. Unfortunately couldn't find them anywhere, the webapp's Trash or the local .trash. Tbh, it's been deleting more files than syncing files to my computer. Seems unusable right now.

**Comment by elbakkayg** [comment]
the problem still exists with the latest version, after synchronization the majority of files are put in the trash..
![Screenshot from 2020-02-24 09 49 56](https://user-images.githubusercontent.com/60742919/75138882-937d7f00-56eb-11ea-84c0-971331e78a26.png)

![Screenshot from 2020-02-24 09 50 29](https://user-images.githubusercontent.com/60742919/75138889-9aa48d00-56eb-11ea-94b5-d6a925f5d7ca.png)

**Comment by bcedu** [comment]
First of all, we apologize for any inconvenience.

Made a change to make VGrive move files on Google Drive to trash instead of deleting them to minimize impact in case of delay errors (https://github.com/bcedu/VGrive/releases/tag/v1.5.0). Detects problems with files with "special" or "non-standard" characters. Tried some characters like "@", "&", "` "," ´ ", and it worked fine but I am having trouble with characters from other alphabets.

**Comment by elementmike** [comment]
deleted some of my files too, with no trace at all ...

**Comment by Arcstur** [comment]
I'm having issues with "#" before names, "_" is working fine and also "á" "ã".

**Comment by bcedu** [comment]
I will check it

**Comment by metal03326** [comment]
Yep, same here - one folder completely went to the .trash with all it's sub-folders. Restoring everything was not an easy task at all. Good thing that I had copy of the folder on an external drive.

**Comment by jazzdup** [comment]
It's trying to delete all my files - saying they're deleted remotely but they're only deleted locally.
This is bad, I'm uninstalling... (ubuntu 20.04)

**Comment by jpbartolomeo** [comment]
I don't understand. Same happened to me but I'm in the version 1.6.1-2, and . There is a new version now but I will now update if it has the same error. Does this problem have a solution?

Arch Linux LTS (5.10.34-1-lts)

**Comment by jesbenmx** [comment]
Same problem, this cliente does not work. ever create a .trash directory with bis space 23gb today and think they are all files, i do not delete nothing
